config/routes.rb

```diff
@@ -280,8 +280,10 @@
   resource :profile, :only => [:edit, :update]
 
   # friendships
-  match "/user/:display_name/make_friend" => "friendships#make_friend", :via => [:get, :post], :as => "make_friend"
-  match "/user/:display_name/remove_friend" => "friendships#remove_friend", :via => [:get, :post], :as => "remove_friend"
+  scope "/user/:display_name" do
+    resource :follow, :only => [:create, :destroy], :path => "follow"
+  end
+  get "/user/:display_name/follow" => "follows#index"
```

Comment:
It's a page that either shows unfollow functionality or follow functionality based on if user already follows `:display_name`. As it is default `GET` method for this functionality (while other ones are `POST` and `DELETE`), I decided to name it `follows#index`. If you have an idea of a better name, I am open to it.

Reply:
Why do you need this page instead of using the standard `:new` and `:edit` pages?
https://guides.rubyonrails.org/v7.2/routing.html#singular-resources

Reply:
I agree with you. Changed `GET /follow` to `GET /follow/edit`. It's much intuitive and consistent.